community/kernel/src/main/java/org/neo4j/kernel/impl/store/StoreFactory.java

```diff
@@ -148,12 +148,6 @@ public NeoStore newNeoStore( boolean allowCreateEmpty )
             return createNeoStore();
         }
 
-        CountsTracker countsTracker = null;
-        if ( storeExists && countsStoreExists() )
-        {
-            countsTracker = newCountsStore();
-        }
-
```

Comment:
The new tracker handles that internally. It doesn't actually open anything until `init()`.